README.rst

```diff
@@ -1,64 +1,241 @@
+
+The managed Jenkinses at Zalando run under the Continuous Delivery team's
+AWS account. The Continuous Delivery team is responsible for
+maintaining the Jenkinses of all other Zalando teams. That allows
+other teams in the company to focus on their `OKR's`_ and not invest
+their time on configuring Jenkins or continuous deployment tools. The
+teams at Zalando enjoy `Jenkins-as-a-Service`_.
+
+Each team at Zalando has their own AWS account. So running the
+`Senza`_ tool within a managed Jenkins job would allow deployments only in
+the Continuous Delivery team's AWS account. To permit different teams
+to deploy in their own AWS account Lizzy was created.
+
+How Lizzy works?
+================
+
+Lizzy consists of a REST API service (also called Agent) and the
+`Lizzy-Client`_ tool.
+
+The Lizzy Agent is a web application that is deployed in the team's
+account and granted access to create new Cloud Formation stacks. All
+requests are expected to be authenticated using `OAuth2`_ Bearer Tokens.
+
+The `Lizzy-Client`_ is a command line tool that mimics the usage of
+the original `Senza`_ tool commands and transforms those commands to HTTP
+requests sent to the REST API of Lizzy Agent.
+
+Who is using Lizzy?
+===================
+
+Lizzy is being used in production at Zalando to delivery high quality
+services to power the `fastest growing`_ online fashion
+retailer store in Europe. If you want to know more about how Lizzy is
+used at Zalando please take a look at our blog post
+`Continuous Delivery pipelines of ghe-backups with Lizzy`_.
+
+How to run Lizzy locally?
+=========================
+
+We provide in this repository a `Dockerfile`_ which you can use to
+build an image locally using the command:
+
+.. code-block:: sh
+
+    $ docker build -t lizzy:dev .
+
+
+.. hint:: The "scm-source.json" file will be missing if you just clone
+          this repository. To know more what is this file, please read
+          the `STUPS documentation`_. We have `tools`_ that can generate
+          this file for you.
+
+After the image build, it will be available within the tag
+"lizzy:dev". You will also need to specify some environment
+variables to be used by Lizzy. Here is an "example.cfg" file:
+
+.. code-block:: cfg
+
+    TOKEN_URL=https://token.auth.example.com/oauth2/access_token
+    TOKENINFO_URL=https://info.auth.example.com/oauth2/tokeninfo
+    ALLOWED_USERS=['robotusername','myusername']
+    DEPLOYER_SCOPE=deployer
+    LOG_LEVEL=DEBUG
+
+It is also necessary to configure the `AWS credentials`_ locally in
+your machine under `~/.aws`. After that you can run the Lizzy image
+with the command:
 
+.. code-block:: sh
+
+    $ docker run --rm -p 8080:8080 --env-file environment.cfg -v ~/.aws:/.aws --name lizzy -u 999 -it lizzy:dev
+
+The application by default will be listening on port `8080`. Usually
+accessible at `http://127.0.0.1:8080`, it depends on how your Docker is
+configured locally. A Swagger/OpenAPI console will be available at
```

Comment:
Do you mean to say
> By default, the application will be accessible on port `8080` & on url `http://127.0.0.1/` though this might vary based on how Docker is configured on the local system?